vm/kvm-unit-tests/runtest.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+. /usr/bin/rhts_environment.sh
+
+export ACCEL=kvm
+unset ARCH
+unset STANDALONE
+cpus=$(grep -c ^processor /proc/cpuinfo)
+
+# test can only run on hardware that supports virtualization 
+if egrep -q '(vmx|svm)' /proc/cpuinfo; then
+    echo "Hardware supports virtualization, proceeding" | tee -a $OUTPUTFILE
+else
+    echo "Skipping test, CPU doesn't support virtualization" | tee -a $OUTPUTFILE
+    rhts-report-result $TEST SKIP $OUTPUTFILE
+    exit
+fi
+
+# test should only run on a system with 1 or more cpus
+if [ "$cpus" > 1 ]; then
+    echo "You have sufficient CPU's to run the test" | tee -a $OUTPUTFILE
+else
+    echo "Skipping test, system requires > 1 CPU" | tee -a $OUTPUTFILE
```

Comment:
Should the `rhts-report-result $TEST SKIP $OUTPUTFILE` be included in this branch too?

Reply:
@veruu your right, I added it